.travis.yml

```diff
@@ -31,5 +31,6 @@ install:
   
 script: 
   # Finally, build fserv.
-  - cd ~/build/f-list/fserv
-  - make all
+  - cd $HOME/build/$TRAVIS_REPO_SLUG
+  - cmake .
```

Comment:
Probably should use a dedicated build directory. In tree building with CMake is discouraged.